scripts/pin_the_slow_way.py

```diff
@@ -70,7 +78,7 @@
           'tk': 'tk 8.5.*',  # 8.5.18
           'vlfeat': 'vlfeat 0.9.20',  # NA
           'xz': 'xz 5.2.*',  # 5.2.2
-          'zlib': 'zlib 1.2.*',  # 1.2.8
+          'zlib': 'zlib 1.2.8',  # 1.2.8
```

Comment:
This unfortunately needs to happen

ref: https://abi-laboratory.pro/tracker/timeline/zlib/

Reply:
@jakirkham why cant we unpin zlib exactly?

Reply:
@jschueller , zlib has a tendency to add symbols on patch versions (major.minor.patch) as can be seen on https://abi-laboratory.pro/tracker/timeline/zlib/
For example ```1.2.9``` adds 8 symbols when compared to ```1.2.8```. If we pin zlib with ```1.2.*``` we allow a package to be built with ```1.2.9``` (and therefore allow it to use the new symbols) but run with ```1.2.8```, possibly causing a runtime error when loading symbols.

Reply:
Ok.
Could we pin to latest instead of 1.2.8 ?

Reply:
I think there is no problem, but it would be good to then rerender and rebuild all feedstocks that depends directly on ```zlib```. conda shows these ones:

```bash
conda search --reverse-dependency zlib --json | jq '. | keys'
[
  "anaconda",
  "boost",
  "cairo",
  "cmake",
  "curl",
  "freetype",
  "git",
  "glib",
  "graphviz",
  "hdf4",
  "hdf5",
  "imaging",
  "libhdfs3",
  "libnetcdf",
  "libpng",
  "libprotobuf",
  "libssh2",
  "libthrift",
  "libtiff",
  "libxml2",
  "nano",
  "netcdf4",
  "nginx",
  "opencv",
  "pcre",
  "pigz",
  "pil",
  "pillow",
  "pkg-config",
  "postgresql",
  "pycurl",
  "pytables",
  "python",
  "qt",
  "r",
  "r-base",
  "r-png",
  "serf",
  "svn",
  "tensorflow",
  "the-silver-searcher",
  "wget"
]
```

Anyway I think @jakirkham or the other core members have a better idea on the implications of this change.

Reply:
wow, I was just curious...